### ISSUE TITLE
Using pseudo class with keyframes produces Fatal error

### ISSUE THREAD

**Issue by ukupat** [sasstools/sass-lint#796]
**What version of Sass Lint are you using?**
v1.8.2

**Please include any relevant parts of your configuration**
We are not using default rules (we have defined our own rules list in config), but anything else is default.

**What did you do?**
We started using CSS Modules but we wanted to use some keyframes globally that were defined in .scss. For that we had to use `:global` pseudo class that takes an argument https://github.com/css-modules/css-modules#exceptions. Something like this:

``` css
@keyframes :global(alert--fadeout-animation) {
    0% { opacity: 1; }
    100% { opacity: 0; }
}
```

This is the only way to use keyframes globally because of https://github.com/css-modules/css-modules/issues/115

**What did you expect to happen?**
We expected to see no linting errors, because using :global pseudo class as a wrapper didn't cause any errors:

``` css
:global {
  .example {...}
}
```

**What actually happened?**
Linting produced Fatal error results:

``` javascript
2:1  error  Please check validity of the block starting from line #2  Fatal
```

**If you're using a IDE plugin have you tried the CLI too?**
Had the same results when using Atom plugin and CLI.

**Comment by DanPurdy** [comment]
HI, this is an issue with the AST sass-lint is using, please report the issue there also. https://github.com/tonyganch/gonzales-pe

**Comment by ukupat** [comment]
I will, thanks!

**Comment by bgriffith** [comment]
This is fixed in the newly released version `v1.11.0`. Please let me know if you still have issues. Thanks!
